enterprise/frontend/src/embedding-sdk/test/snippets/tsconfig.snippets.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "../../../../../../tsconfig.base.json",
+  "compilerOptions": {
+    "paths": {
+      "*": ["../../../../../../enterprise/frontend/src/*"],
+      "@metabase/embedding-sdk-react": ["../../../../../../resources/embedding-sdk"]
+    },
+    "noImplicitAny": false,
+    "allowJs": true,
+    "checkJs": true
+  },
+  "include": ["./"]
+}
```

Comment:
I can put it at the root, but because it is used in these tests only, right now it is here.